modules/learn/pages/services-and-indexes/indexes/index_pushdowns.adoc

```diff
@@ -318,731 +199,333 @@ Pagination in N1QL queries is achieved by using the LIMIT and OFFSET clauses, an
+
+[[example-idx-3]]
+.Descending Sort by String Field
+====
+Descending variation of <<example-idx-1>>.
+
+.Index
+[source,n1ql]
 ----
-C3: CREATE INDEX def_city ON `travel-sample`(`city` DESC);
+CREATE INDEX def_city_desc ON `travel-sample`(`city` DESC);
 ----
 
-*Q3*: Descending variation of Q1.
-
-[source,json]
+.Query
+[source,n1ql]
 ----
 EXPLAIN SELECT city FROM `travel-sample`
 WHERE city LIKE "San%"
-ORDER BY meta().id;
+ORDER BY city DESC;
+----
 
-Result:
-{
-  "plan": {
-    "#operator": "Sequence",
-    "~children": [
-      {
-        "#operator": "Sequence",
-        "~children": [
-          {
-            "#operator": "IndexScan2",
-            "covers": [
-              "cover ((`travel-sample`.`city`))",
-              "cover ((meta(`travel-sample`).`id`))"
-            ],
-            "index": "def_city",
-            "index_id": "fd399cb179e9ab0a",
-            "keyspace": "travel-sample",
-            "namespace": "default",
-            "spans": [
-              {
-                "exact": true,
-                "range": [
-                  {
-                    "high": "\"Sao\"",
-                    "inclusion": 1,
-                    "low": "\"San\""
-                  }
-                ]
-              }
-            ],
-            "using": "gsi"
-          },
-          {
-            "#operator": "Parallel",
-            "~child": {
-              "#operator": "Sequence",
-              "~children": [
-                {
-                  "#operator": "Filter",
-                  "condition": "(cover ((`travel-sample`.`city`)) like \"San%\")"
-                },
-                {
-                  "#operator": "InitialProject",
-                  "result_terms": [
-                    {
-                      "expr": "cover ((`travel-sample`.`city`))"
-                    }
-                  ]
-                }
-              ]
-            }
-          }
-        ]
-      },
-      {
-        "#operator": "Order",
-        "sort_terms": [
-          {
-            "expr": "cover ((meta(`travel-sample`).`id`))"
-          }
-        ]
-      },
-      {
-        "#operator": "FinalProject"
-      }
-    ]
-  },
-  "text": "SELECT city FROM `travel-sample` \nWHERE city LIKE \"San%\"\nORDER BY meta().id;"
-}
+.Result
+[source,json]
+----
+include::example$services-and-indexes/indexes/pushdown-idx-3.json[tag=index]
+...
+include::example$services-and-indexes/indexes/pushdown-idx-3.json[tag=query]
+...
 ----
+====
 
 == Operator Pushdowns
 
-The N1QL query engine tries to avoid unnecessary processing operators such as MIN(), MAX(), and COUNT(), which can be processed by Indexer much more efficiently.
-In such cases, N1QL pushes down necessary hints or options to the Indexer to process the following operators.
+The Query engine tries to avoid unnecessary processing operators such as MIN(), MAX(), and COUNT(), which can be processed by Indexer much more efficiently.
```

Comment:
```suggestion
The Query engine tries to avoid unnecessary processing operators such as MIN(), MAX(), and COUNT(), which can be processed by the indexer much more efficiently.
```